src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserManagementResourceIntTest.kt

```diff
@@ -414,7 +414,26 @@ class UserManagementResourceIntTest : IntegrationTestBase() {
         .expectStatus().is4xxClientError
         .expectBody()
         .jsonPath("userMessage")
-        .isEqualTo("Validation failure: Last name must consist of alphabetical characters only and a max 35 chars")
+        .isEqualTo("Validation failure: Last name must consist of alphabetical characters or an apostrophe only and a max 35 chars")
```

Comment:
Wondered if the creates should also do this kind of test? to test error messages ?